api/helpers_test.go

```diff
@@ -0,0 +1,51 @@
+package api
+
+import (
+	"testing"
+
+	"github.com/netlify/gotrue/conf"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type HelpersTestSuite struct {
+	suite.Suite
+	API    *API
+	Config *conf.Configuration
+}
+
+func (ts *HelpersTestSuite) SetupSuite() {
+}
+
+func (ts *HelpersTestSuite) TearDownSuite() {
+}
+
+func (ts *HelpersTestSuite) SetupTest() {
+	api, config, err := NewAPIFromConfigFile("test.env", "v1")
+	require.NoError(ts.T(), err)
+
+	ts.API = api
+	ts.Config = config
+}
+
+func TestHelpers(t *testing.T) {
+	suite.Run(t, new(HelpersTestSuite))
+}
+
+func (ts *HelpersTestSuite) TestHelpers_IsPasswordValid() {
+	require.False(ts.T(), ts.API.isPasswordValid(""), "password should not be valid")
```

Comment:
I'd use `assert` not `require`, the latter will cancel the rest of the run. 

And `ts.Assert().False(....)` is cleaner https://godoc.org/github.com/stretchr/testify/suite#Suite.Assert